ui/pages/confirmations/components/confirm/info/token-transfer/name-or-address-display.tsx

```diff
@@ -0,0 +1,59 @@
+import { NameType } from '@metamask/name-controller';
+import React, { useState } from 'react';
+import { useSelector } from 'react-redux';
+import { useFallbackDisplayName } from '../../../../../../components/app/confirm/info/row/hook';
+import NicknamePopovers from '../../../../../../components/app/modals/nickname-popovers';
+import Name from '../../../../../../components/app/name';
+import {
+  AvatarAccount,
+  AvatarAccountSize,
+  Box,
+  Text,
+} from '../../../../../../components/component-library';
+import {
+  AlignItems,
+  BorderColor,
+  Display,
+  FlexDirection,
+  TextColor,
+} from '../../../../../../helpers/constants/design-system';
+import { getPetnamesEnabled } from '../../../../../../selectors';
+
+export const NameOrAddressDisplay = ({ address }: { address: string }) => {
+  const isPetNamesEnabled = useSelector(getPetnamesEnabled);
+  const { displayName, hexAddress } = useFallbackDisplayName(address);
+  const [isNicknamePopoverShown, setIsNicknamePopoverShown] = useState(false);
+  const handleDisplayNameClick = () => setIsNicknamePopoverShown(true);
+  const onCloseHandler = () => setIsNicknamePopoverShown(false);
+
+  if (isPetNamesEnabled) {
```

Comment:
I recall we had plans to remove the toggle for petnames so this won't be needed soon, is it simpler to force this now and remove the alternate code path and maybe even this entire component?